app/views/issues/_issues_table.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="table-responsive">
+  <table class="table">
+    <thead class="thead-light">
+      <tr>
+        <th><%= t('shared.sq_num_field', model: Issue.model_name.human) %></th>
```

Comment:
Issue.to_s

Reply:
This is a way to include internationalization as done in en.yml activerecord models internationalization